pom.xml

```diff
@@ -64,6 +64,12 @@
             <artifactId>lombok</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <scope>runtime</scope>
+            <version>42.6.2</version>
+        </dependency>
```

Comment:
**question**

Why moving this from the model package ?
Postgres is direclty used by the repository class.

Reply:
An older version of Postgres is pulled by openbas-api and openbas-framework because of other dependencies. Setting it in the root pom.xml will make sure we override the older version we don't want.

Reply:
Ok understood
Thanks for the explanation.